app/src/test/kotlin/com/intive/picover/main/viewmodel/MainViewModelTest.kt

```diff
@@ -15,7 +18,8 @@ import kotlinx.coroutines.flow.flowOf
 class MainViewModelTest : ShouldSpec(
 	{
 		val authRepository: AuthRepository = mockk()
-		val tested by lazy { MainViewModel(authRepository) }
+		val launcher: Launcher = mockk(relaxUnitFun = true)
+		fun tested() = MainViewModel(authRepository, launcher, mockk())
```

Comment:
Wouldn't be better to use property, as before, with custom getter?

Reply:
That was also my initial idea, but it's available only on class properties, and here we are in scope of lambda from ShouldSpec, which has some downsides like this one 😄

Reply:
Good to know 😄  Maybe we can use `isolationMode = IsolationMode.InstancePerTest` and keep `tested` as previosuly?

Reply:
That should work, thanks for the hint! 😄